aleph/settings.py

```diff
@@ -5,10 +5,15 @@
 # defaults.
 import json
 import os
-from servicelayer import env
+from datetime import timedelta
+from json.decoder import JSONDecodeError
 from urllib.parse import urlparse
+
+import structlog
 from flask_babel import lazy_gettext
-from datetime import timedelta
+from servicelayer import env
```

Comment:
Is this required?

Reply:
No not actually, but that was consistent with the style of the file before I first started editing it